src/Networking/NexusMods.Networking.HttpDownloader/AdvancedHttpDownloader.cs

```diff
@@ -46,7 +46,7 @@ public class AdvancedHttpDownloader : IHttpDownloader
         /// </summary>
         private readonly Size _chunkSize = Size.MB * 128;
 
-        private const int MaxRetries = 4;
+        private const int MaxRetries = 16;
```

Comment:
We can use the `CI` environment variable to check whether we're running in CI or not. Instead of setting the max retries to `16` for everyone, we can set it to `4` for users and `16` for CI.

Reply:
We can, but is there a problem with having a higher retry count on all platforms?

Reply:
Probably not. It's just something to keep in mind when tuning other settings for CI specifically.